lib/mix/tasks/phoenix_slime.gen.html.ex

```diff
@@ -0,0 +1,147 @@
+defmodule Mix.Tasks.PhoenixSlime.Gen.Html do
+  use Mix.Task
+
+  @shortdoc "Generates controller, model and views for an HTML based resource"
```

Comment:
I agree it's funky which is why I appreciate what you're saying.  I just can't shake the feeling that it should be clearer what you're going to be getting.  I don't think `phoenix_slime` immediately jumps out to everyone.

Your suggested description is a good compromise :+1: